runbot/models/build_config.py

```diff
@@ -884,10 +884,9 @@ def _log_end(self, build):
 
         kwargs = dict(message='Step %s finished in %s' % (self.name, s2human(build.job_time)))
         if self.job_type == 'install_odoo':
-            kwargs['message'] += ' $$fa-download$$'
             db_suffix = build.params_id.config_data.get('db_name') or (build.params_id.dump_db.db_suffix if not self.create_db else False) or self.db_name
-            kwargs['path'] = '%s%s-%s.zip' % (build._http_log_url(), build.dest, db_suffix)
-            kwargs['log_type'] = 'link'
+            kwargs['message'] += ' [@icon-download](%s)' % ('%s%s-%s.zip' % (build._http_log_url(), build.dest, db_suffix))
+            kwargs['log_type'] = 'markdown'
         build._log('', **kwargs)
```

Comment:
In the case of markdown logs, _http_log_url, dest, suffix should ideally be passed as arguments to the  build._log method so that it is escaped. This will avoid to potentially escape parts of the _http_log_url

In this case it is not a big issue I think but it would be cleaner

```suggestion
        build._log('',  ' [@icon-download](%s)' % ('%s%s-%s.zip' , build._http_log_url(), build.dest, db_suffix, log_type= 'markdown')
```